orga/tests/acceptance/join-and-login-test.js

```diff
@@ -0,0 +1,208 @@
+import { clickByName, fillByLabel, visit } from '@1024pix/ember-testing-library';
+import { click, currentURL } from '@ember/test-helpers';
+import setupMirage from 'ember-cli-mirage/test-support/setup-mirage';
+import { t } from 'ember-intl/test-support';
+import { setupApplicationTest } from 'ember-qunit';
+import { currentSession } from 'ember-simple-auth/test-support';
+import { Response } from 'miragejs';
+import { module, test } from 'qunit';
+
+import setupIntl from '../helpers/setup-intl';
+import {
+  createPrescriberByUser,
+  createUserWithMembership,
+  createUserWithMembershipAndTermsOfServiceAccepted,
+} from '../helpers/test-init';
+
+module('Acceptance | join and login', function (hooks) {
+  setupApplicationTest(hooks);
+  setupMirage(hooks);
+  setupIntl(hooks);
+
+  hooks.beforeEach(function () {
+    server.create('feature-toggle', { id: 0, usePixOrgaNewAuthDesign: true });
+  });
+
+  module('When the invitation is valid', function (hooks) {
+    let invitationId;
+    let invitationCode;
+
+    hooks.beforeEach(() => {
+      invitationCode = 'ABCD';
+      invitationId = server.create('organizationInvitation', {
+        status: 'pending',
+        code: invitationCode,
+        organizationName: 'College BRO & Evil Associates',
+      }).id;
+    });
+
+    test('authenticates with invitation and access the organization', async function (assert) {
```

Comment:
Suggestion de formulation pour cet intitulé de test, mais je ne suis pas sûr de moi :
```suggestion
    test('authenticates with invitation and joins the organization', async function (assert) {
```